jipdate.py

```diff
@@ -294,40 +301,55 @@ def parse_status_file(jira, filename):
 
             try:
                 issue = jira.issue(myissue)
+                issue_comments.append((issue, "", ""))
             except  Exception as e:
                 if 'Issue Does Not Exist' in e.text:
                     print ('[{}] :  {}'.format(myissue, e.text))
                     validissue = False
 
-            if validissue:
-                issue_comments.append((myissue, ""))
         # Stop parsing entirely.  This needs to be placed before regex_stop
         # or the .* will match and [FIN] won't be processed
         elif re.search(regex_fin, line):
             break
         # If we have non-JIRA issue tags, stop parsing until we find a valid tag
         elif re.search(regex_stop, line):
                 validissue = False
+        elif re.search(regex_status, line):
+            transition = line[8:].strip()
```

Comment:
You are stripping away "`Status: `" here. Wouldn't you be able to avoid this by leverage the `(.+)` from the regexp?